integrations/apollo/src/index.ts

```diff
@@ -23,8 +22,7 @@ export type NhostApolloClientOptions = {
   fetchPolicy?: WatchQueryFetchPolicy
   connectToDevTools?: boolean
   cache?: InMemoryCache
-  onError?: RequestHandler
-  link?: ApolloClient<any>['link']
+  generateLinks?: Function
```

Comment:
Is there a type for links in general that we can use for this function?

I'm thinking of something like this:
```ts
generateLinks?: (...links: ApolloLink[]) => void
```

Reply:
Sounds good, I'll make the change